examples/data-objects/focus-tracker/src/FocusTracker.ts

```diff
@@ -0,0 +1,165 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { EventEmitter } from "events";
+import { IEvent } from "@fluidframework/common-definitions";
+import {
+    DataObject,
+    DataObjectFactory,
+    IMember,
+    IServiceAudience,
+} from "@fluid-experimental/fluid-framework";
+import { IInboundSignalMessage } from "@fluidframework/runtime-definitions";
+
+export interface IFocusTrackerEvents extends IEvent {
+    (event: "focusChanged", listener: () => void): void;
+}
+
+/**
+ * Data object example of using the audience with signals to track focus
+ * state of connected clients without writing changes to a DDS
+ */
+// eslint-disable-next-line @typescript-eslint/ban-types
+export class FocusTracker extends DataObject<{}, undefined, IFocusTrackerEvents> implements EventEmitter {
```

Comment:
is there an advantage to DataObject vs just using EventEmitter

Reply:
The signal API is only available from inside a DO so we're kind of stuck there.  The signal DO helper would help expose that elsewhere and also provide some controls/guardrails on its usage (coming soon™?)

Reply:
oh, that certainly complicates using signals in some of our other demo code (which is trying to avoid using data objects). Certainly would be helpful to tease that helper out